src/python/repo.py

```diff
@@ -93,6 +93,36 @@ def clone_repo(repo_slug: str, repo_dir: Path) -> git.repo.Repo:
         ) from None
 
 
+# Alternative clone repo method that returns git repo object for diff3 scripts
```

Comment:
How is it different?  The documentation is identical.

Reply:
Fixed.